scripts/build_schemas.sh

```diff
@@ -2,18 +2,25 @@
 
 set -e
 
-# Build schemas for each region
-for region_code in GB-WLS GB-ENG; do
-    SOURCE_FILE="data-source/jsonnet/census_individual.jsonnet"
+# Build schema for each region
+for region_code in GB-WLS GB-ENG GB-NIR; do
     # Lowercase the region code and replace '-' with '_'
     FORMATTED_REGION_CODE=$(echo "${region_code}" | tr '[:upper:]' '[:lower:]' | tr - _)
     DESTINATION_FILE="data-source/jsonnet/census_individual_${FORMATTED_REGION_CODE}.json"
 
-    jsonnet --tla-str region_code=${region_code} --tla-str census_date="2019-09-01" "${SOURCE_FILE}" > "${DESTINATION_FILE}"
+    if [[ "$region_code" = "GB-NIR" ]]; then
+        NI_SOURCE_FILE="data-source/jsonnet/nir/census_individual.jsonnet"
+        jsonnet --tla-str region_code=${region_code} "${NI_SOURCE_FILE}" > "${DESTINATION_FILE}"
+    else
+        NON_NI_SOURCE_FILE="data-source/jsonnet/eng-wls/census_individual.jsonnet"
```

Comment:
- `NON_NI_SOURCE_FILE` is not a good name, why not just use `SOURCE_FILE` for both?
- I think we should rename the directories to `england-wales` and `northern-ireland`

Reply:
Didn't have an `if` statement to begin with so made sense then but didn't refactor after. Have sorted this now. And also renamed the directories.